src/main/resources/attackvectors/ErrorBasedSQLInjectionPayload.properties

```diff
@@ -0,0 +1,21 @@
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=By sending a malicious query into the input fields we should receive a SQL or database syntax error. It will allow us to get information about table names or even query structure.<br/>\
+ How to exploit this level?<br/>\
+ Insert this in textbox<br/>\
+ # <br/>\
```

Comment:
For this all the levels are exploited by # actually lets tell the specific payload for each level like we have for other vulnerabilities.